packages/api-v2/src/hooks/useCFDCompareAccounts.tsx

```diff
@@ -12,6 +12,14 @@ export const MARKET_TYPE = {
     SYNTHETIC: 'synthetic',
 } as const;
 
+// Remove the hardcoded values and use the values from the API once it's ready
+export const PRODUCT = {
+    CTRADER: 'ctrader',
+    DERIVX: 'derivx',
+    SWAPFREE: 'swap_free',
```

Comment:
are we using these?

Reply:
yes. @aizad-deriv commented to add it avoid string 'zero_spread' in the file.